Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/PasswordVaultSamplePage.xaml

```diff
@@ -0,0 +1,58 @@
+﻿<Page x:Class="Uno.Gallery.Views.Samples.PasswordVaultSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:samples="using:Uno.Gallery.Views.Samples"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:android="http://uno.ui/android"
+	  xmlns:ios="http://uno.ui/ios"
+	  xmlns:macos="http://uno.ui/macos"
+	  xmlns:skia="http://uno.ui/skia"
+	  xmlns:wasm="http://uno.ui/wasm"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  mc:Ignorable="d android ios macos skia wasm"
+	  x:Name="page">
+
+    <Grid Background="{StaticResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout>
+			<local:SamplePageLayout.FluentTemplate>
```

Comment:
We should have a `DesignAgnosticTemplate`, as we do in the [Uno.Toolkit Sample App](https://github.com/unoplatform/uno.toolkit.ui/blob/17369bd6f9b16288c74e5e28716e1e69c28e099e/samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Controls/SamplePageLayout.Properties.cs#L153-L167)

Not a blocker for this PR but something that can be updated to use in the future

Reply:
No need , reported as separate issue